src/components/Modals/ConfirmTransaction/index.tsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import 'styled-components/macro';
+import { colors, GU } from 'components/theme';
+
+import { ReactComponent as CoinSvg } from 'assets/icons/coin.svg';
+import { ReactComponent as CloseSvg } from 'assets/icons/close.svg';
+import { ReactComponent as CopySvg } from 'assets/icons/copy.svg';
+import { ReactComponent as DepositButtonActiveSvg } from 'assets/icons/deposit_button_active.svg';
+import { ReactComponent as FarmSvg } from 'assets/icons/farm.svg';
+import { ReactComponent as MultiplierSvg } from 'assets/icons/multiplier.svg';
+import { ReactComponent as RewardsGraphSvg } from 'assets/icons/rewards_graph.svg';
+import { ReactComponent as WithdrawButtonActiveSvg } from 'assets/icons/withdraw_button_active.svg';
```

Comment:
Did we run these SVGs through an optimizer tool? If not, we should!

Reply:
Consider them optimized! Fixed in 384a31f3236bfb0c116da635e532d0e2f0383fa1